index.js

```diff
@@ -175,16 +176,18 @@ function isContributionInTimeRange(createdAt, startMoment, endMoment) {
     );
 }
 
-function filterContributorByTime(idObject, moments) {
+function didTheyQualify(idObject, moments) {
     const startMoment = moments[0];
     const endMoment = moments[1];
+    let numberOfValidContributions = 0;
```

Comment:
Minor terminology thing, but the other contributions aren't "invalid", so much as not relevant or qualifying. Should we rename this to numberOfQualifyingContributions?

Reply:
Sure :-)